app/views/shared/_header.html.erb

```diff
@@ -1,8 +1,10 @@
 <header role="banner" id="global-header">
   <div class="wrapper">
-    <%= link_to(home_path, :title => 'Go to the petitions homepage', :tabindex => 1) do %>
-      <span class="graphic graphic-portcullis-white" aria-hidden="true"></span> Petitions
-    <% end %>
+    <h1>
```

Comment:
Unfortunately we only want this to be an `h1` when it's the home page. On other pages we have good `h1`s already. This was not fully explained in the story, my bad.